pyxrf/model/draw_image_rgb.py

```diff
@@ -290,17 +309,69 @@ def _compute_equal_axes_ranges(x_min, x_max, y_min, y_max):
 
             return x_axis_min, x_axis_max, y_axis_min, y_axis_max
 
-        # Set equal ranges for the axes data
-        yd, xd = selected_data.shape[1], selected_data.shape[2]
-        xd_min, xd_max, yd_min, yd_max = 0, xd, 0, yd
-        # Select minimum range for data
-        if (yd <= math.floor(xd / 100)) and (xd >= 200):
-            yd_min, yd_max = -math.floor(xd / 200), math.ceil(xd / 200)
-        if (xd <= math.floor(yd / 100)) and (yd >= 200):
-            xd_min, xd_max = -math.floor(yd / 200), math.ceil(yd / 200)
+        def _adjust_data_range__min_ratio(c_min, c_max, c_axis_range, *, min_ratio=0.01):
```

Comment:
I don't think the double underscore in the name is in any naming convention.
```suggestion
        def _adjust_data_range_min_ratio(c_min, c_max, c_axis_range, *, min_ratio=0.01):
```

Reply:
Do you think it's a violation of naming convention? Typically naming convention restricts the number of underscores at the beginning of the name. I rarely use double underscores to separate two parts of the name. If you insist on changing, I guess the right name will be `adjust_data_range_using_min_ratio` or something like this, but it becomes long.